apps/molgenis-viz/src/views/home-view.vue

```diff
@@ -41,59 +41,74 @@
-        </LinkCard>
-        <LinkCard imageSrc="highlights-header.jpg">
-          <router-link :to="{ name: 'data-highlights' }">
-            Data Highlights
-          </router-link>
-        </LinkCard>
-        <LinkCard imageSrc="map-header.jpg">
-          <router-link :to="{ name: 'geo-mercator' }">
-            GeoMercator
-          </router-link>
-        </LinkCard>
-        <LinkCard imageSrc="grouped-column-chart-header.jpg">
-          <router-link :to="{ name: 'grouped-column-chart' }">
-            Grouped Column Chart
-          </router-link>
-        </LinkCard>
-        <LinkCard imageSrc="legend-header.jpg">
-          <router-link :to="{ name: 'chart-legend' }"> Legends </router-link>
-        </LinkCard>
-        <LinkCard imageSrc="pie-chart-header.jpg">
-          <router-link :to="{ name: 'pie-chart' }"> Pie Chart </router-link>
-        </LinkCard>
-        <LinkCard imageSrc="pie-chart-header.jpg">
-          <router-link :to="{ name: 'pie-chart-2' }"> Pie Chart2 </router-link>
-        </LinkCard>
-        <LinkCard imageSrc="scatter-plot-header.jpg">
-          <router-link :to="{ name: 'scatter-plot' }">
-            Scatter Plot
-          </router-link>
-        </LinkCard>
-        <LinkCard imageSrc="gauge-chart-header.jpg">
-          <router-link :to="{ name: 'progress-charts' }">
-            Progress Charts
-          </router-link>
-        </LinkCard>
-      </div>
+      <QuickLinks
+        v-else
+        id="visualisation-links"
+        :data="[
+          {
+            to: 'bar-chart',
+            label: 'Bar Chart',
+            image: 'bar-chart-header.jpg',
+          },
+          {
+            to: 'column-chart',
+            label: 'Column Chart',
+            image: 'column-chart-header.jpg',
+          },
+          { to: 'datatable', label: 'Data Table', image: 'table-header.jpg' },
+          {
+            to: 'data-highlights',
+            label: 'Data Highlights',
+            image: 'highlights-header.jpg',
+          },
+          {
+            to: 'geo-mercator',
+            label: 'Geo Mercator',
+            image: 'map-header.jpg',
+          },
+          {
+            to: 'grouped-column-chart',
+            label: 'Grouped Column Chart',
+            image: 'grouped-column-chart-header.jpg',
+          },
+          { to: 'chart-legend', label: 'Legends', image: 'legend-header.jpg' },
+          {
+            to: 'pie-chart',
+            label: 'Pie Chart',
+            image: 'pie-chart-header.jpg',
+          },
+          {
+            to: 'pie-chart-2',
+            label: 'Pie Chart 2',
+            image: 'pie-chart-header.jpg',
+          },
+          {
+            to: 'scatter-plot',
+            label: 'Scatter Plot',
+            image: 'scatter-plot-header.jpg',
+          },
+          {
+            to: 'progress-charts',
+            label: 'Progress Charts',
+            image: 'gauge-chart-header.jpg',
```

Comment:
I think data this large should be defined in the script part of the file.

Reply:
Good idea! I've moved this to the script section